src/metrics.rs

```diff
@@ -104,13 +104,24 @@ struct Stats {
     fds: usize,
 }
 
+#[cfg(feature="sysconf")]
+fn ticks_per_second() -> Result<f64> {
+    sysconf::raw::sysconf(sysconf::raw::SysconfVariable::ScClkTck)
+        .map_err(|_| "failed to get _SC_CLK_TCK".into())
```

Comment:
This loses valuable error information that could be important for debugging. Please keep the error somehow (displaying it is OK).